### PR TITLE
Turn student comments to array

### LINKED ISSUE #80: PATCH Student Comments - Array Handling
Allow FE to have multiple comments and send to BE in array.
Join array and store in DB with separating character ("/2C/")
Parse on separating character and put in array when sent back out in API request

### LINKED ISSUE #81: Instructor Students - Update Order
return results ordered by first name then last name

### PR DESCRIPTION
**Pull request by Gvieve**
### Update / Remove
* get_students method on User to use AR and order results by first name then last name
* allow student comments to be received and sent out in an array format
  * array is joined on BE before saving in DB using "/2C/" as our special character to denote each element
* API contract to reflect the changes above
### Why?
* User empathy for instructor, better to display list in alphabetical order
* UX preference by FE to allow user to have list of comments rather than just a single string

Close #80 
close #81

### CODE REVIEW

File: app/controllers/api/v1/student_projects_controller.rb [hunk 1]
```diff
@@ -26,7 +26,13 @@ def required_update
     [:student_id, :id, :student_comments]
   end
 
-  def student_comments_params
-    params.permit(:student_comments)
+  def student_comments
```
Comment (jequeza) [inline app/controllers/api/v1/student_projects_controller.rb:29]: pretty cool stuff so much learning!

File: app/models/user.rb [hunk 2]
```diff
@@ -23,10 +23,12 @@ class User < ApplicationRecord
   has_one :user_profile
 
   def get_students(mod_num)
-    students.map do |student|
-      student.user_profile if student.user_profile.current_mod == mod_num
-    end.compact
+    students
```
Comment (jequeza) [inline app/models/user.rb:26]: good ol AR

Comment (jequeza) [reply app/models/user.rb:26]: with some SQL

**Review by jequeza — APPROVED** [review]
Great Job Dev!

File: app/poros/student_projects.rb [hunk 3]
```diff
@@ -33,4 +33,8 @@ def get_project_feedback(data)
       }
     end
   end
+
+  def format_comments(data)
+    data.split("/2C/").to_a if !data.nil?
```
Comment (b-enji-cmd) [inline app/poros/student_projects.rb:38]: Awesome guard!

**Review by b-enji-cmd — APPROVED** [review]
LGTM! You're amazing

**Pull request merged**